src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -410,13 +475,35 @@ public void periodic() {
 		angleSetpointEntry.setDouble(angleSetpoint);
 		launcherFlywheelSetpointEntry.setDouble(rpmSetpoint);
 
-		// sanity check the pivot encoder
+		// PIVOT ENCODER SANITY CHECKS
+		// compares the relative encoder angle vs the absolute encoder angle
+		if (relativeEncoderStartPosition.isPresent()
+				&& Math.abs(getAngle() - getAngleOneMotorAngle()) <= ENCODER_DIFFERENCE_TOLERANCE) {
```

Comment:
The `<=` seems wrong, as that would indicate that things are in sync (as the difference is small).